### PR TITLE
D8 nid 354

### PR DESCRIPTION
**Pull request by DuttonMa**
Add GP reports, see also https://github.com/dof-dss/nidirect-site-modules/pull/228

### CODE REVIEW

File: config/sync/views.view.gp_list.yml [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+      display_extenders: {  }
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+      tags: {  }
+  page_1:
```
Comment (johangant) [inline config/sync/views.view.gp_list.yml:625]: Probably worth renaming the default 'page_1' display ID to something more descriptive

File: config/sync/views.view.gp_practice_list.yml [hunk 2]
```diff
@@ -0,0 +1,2073 @@
...
+        - 'config:field.storage.node.field_gp_appointments'
+        - 'config:field.storage.node.field_gp_lcg'
+        - 'config:field.storage.node.field_gp_partnership_no'
+        - 'config:field.storage.node.field_gp_practice_lead'
+        - 'config:field.storage.node.field_gp_practice_member'
+        - 'config:field.storage.node.field_gp_practice_no'
+        - 'config:field.storage.node.field_gp_prescriptions'
+        - 'config:field.storage.node.field_gp_surgery_name'
+        - 'config:field.storage.node.field_telephone'
+  page_1:
```
Comment (johangant) [inline config/sync/views.view.gp_practice_list.yml:2029]: Rename to something more descriptive?

**Review by johangant — CHANGES_REQUESTED** [review]
Looks good, although where possible we usually rename the default views display machine names to be something descriptive rather than keep their default (page_1/block_1) display ids.

**Review by omahm — APPROVED** [review]
LGTM

**Comment by omahm** [comment]
Sorry, one other thing, it doesn't look like these Views are restricted to the role access in the ticket

**Pull request closed without merging**